shell/platform/darwin/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -432,6 +449,7 @@ - (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
   [accessibility_channel_.get() setMessageHandler:^(id message, FlutterReply reply) {
     HandleEvent((NSDictionary*)message);
   }];
+  _previous_route_ = nil;
```

Comment:
Release before setting to nil?

Reply:
I think this is actually just wrong, moved to destructor